build/lib/common.sh

```diff
@@ -67,7 +67,71 @@ function build::common::generate_shasum() {
   cd -
 }
 
-# TODO: replace with go licenses tool which was used to create ATTRIBUTION.txt files
+
+function build::gather_licenses_new() {
```

Comment:
Doing this change in multiple PRs so introducing temporary _new function.  In my last PR changing the license handling across all projects, I will rename this and remove the old function.